tests/test_mpi.py

```diff
@@ -801,7 +801,7 @@ def test_trivial_eq_2d(self):
             assert np.all(f.data_ro_domain[0, :-1, -1:] == side)
             assert np.all(f.data_ro_domain[0, -1:, :-1] == side)
 
-    @pytest.mark.parallel(mode=[(8, 'basic'), (8, 'diag'), (8, 'overlap'),
+    @pytest.mark.parallel(mode=[(8, 'basic'), (8, 'basic2'), (8, 'diag'), (8, 'overlap'),
```

Comment:
I would drop `basic` here and below... or `overlap` perhaps. These tests can be quite expensive